master/getting-started/mesos/installation/dc-os/framework.md

```diff
@@ -1,41 +1,19 @@
 ---
-title: Framework Install Walkthrough
+title: Calico-DC/OS Install Walkthrough
```

Comment:
This title doesn't match the sidebar entry (Simple Framework Install)

I think both should be changed to "DC/OS Universe Package"